README.md

```diff
@@ -102,6 +102,29 @@ Array<{
 };
 ```
 
+### `/api/getPoolInfo`
+
+#### Input
+
+```js
+{
+  poolMetaDataHashes: Array<string> //hash ids of pool meta data.
+};
+```
+
+#### Output
+
+```js
+{
+  [poolMetaDataHash: string]: {| name: string,
+                                 description: string,
+                                 ticker: string,
+                                 homepage: string,
+                                 extended: string
```

Comment:
what is "extended"

Reply:
I don't know.  actually, upon further inspection, the docs are wrong.  `extended` only sometimes appears.  compare 
` curl https://stage-smash.yoroiwallet.com/api/v1/metadata/40a80b5bfc4dc1ba1bf94a071875c78ee0a563dddef724f5ed21b264d3274a07` with ` curl https://stage-smash.yoroiwallet.com/api/v1/metadata/ca7d12decf886e31f5226b5946c62edc81a7e40af95ce7cd6465122e309d5626`

Reply:
It looks like the extra url param for extended info that doesn't fit the onchain structure